src/test/java/com/volunteer/enterprise/EnterpriseApplicationTests.java

```diff
@@ -20,8 +21,19 @@ void confirmJonSnow_outputsJonSnow() {
         assertEquals("Drogon145@gmail.com", volunteer.getVolunteerEmail());
         assertEquals("Jon Snow", volunteer.getVolunteerUser());
         assertEquals("Passw0rd@01", volunteer.getVolunteerPass());
+    }
 
-
+    @Test
+    void confirmCompanyGoogle_outputsCompanyGoogle() {
+        Company company = new Company();
+        company.setCompanyID(600673);
+        company.setCompanyEmail("owner@google.com");
+        company.setCompanyUser("Sergey Page");
+        company.setCompanyPass("Y@hoo_Suck$");
+        assertEquals(600673, company.getCompanyID());
+        assertEquals("owner@google.com", company.getCompanyEmail());
+        assertEquals("Sergey Page", company.getCompanyUser());
+        assertEquals("Y@hoo_Suck$", company.getCompanyPass());
```

Comment:
Defined a test for `Company` DTO to reduce lack of unit tests, a SonarQube Deadly Sin. [Reference](https://docs.sonarqube.org/display/SONARQUBE45/Lack+of+Unit+Tests)